src/main/java/store/converter/StoreConverter.java

```diff
@@ -0,0 +1,83 @@
+package store.converter;
+
+import store.dto.CurrentStrockDTO;
+import store.dto.PaymentDTO;
+import store.dto.ProductDTO;
+import store.dto.ReceiptDTO;
+import store.enums.ErrorMessage;
+import store.model.Payment;
+import store.model.Product;
+import store.model.Receipt;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StoreConverter {
+    private static final String PRODUCT_REGEX = "^\\[(\\S+)-(\\d+)\\]$";
+
+    public CurrentStrockDTO convertToCurrentProductDTO(List<Product> products) {
+        List<ProductDTO> productDTOS = new ArrayList<>();
+        for (Product product : products) {
+            String name = product.getName();
+            Integer price = product.getPrice();
+            Integer quantity = product.getQuantity();
+            String promotion = product.getPromotion();
+            productDTOS.add(new ProductDTO(name, price, quantity, promotion));
+        }
+
+        return new CurrentStrockDTO(productDTOS);
```

Comment:
아래 코드처럼 stream으로 변환하는건 어떨까요?!

```suggestion
        List<ProductDTO> productDTOS = products.stream()
                .map(product -> new ProductDTO(
                        product.getName(),
                        product.getPrice(),
                        product.getQuantity(),
                        product.getPromotion()))
                .collect(Collectors.toList());

        return new CurrentStrockDTO(productDTOS);
```